net-mgmt/collectd/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=            collectd
+PLUGIN_VERSION=         0.1
+PLUGIN_COMMENT=         collectd agent
```

Comment:
"Collect system and application performance metrics periodically" via https://collectd.org/